<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> 
<html> <!--<![endif]-->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="description" content="RECOMB 2023 - Bilkent University, Ankara">
  
  <meta name="author" content="Themefisher.com">

  <title>RECOMB 2023 | POSTER GUIDELINES</title>

  <link rel="alternate" href="http://recomb2023.bilkent.edu.tr" hreflang="en-gb" />
  <link rel="alternate" href="http://recomb2023.bilkent.edu.tr" hreflang="en-us" />

<!-- Mobile Specific Meta
  ================================================== -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <!-- Favicon -->
  <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png" />
  
  <!-- CSS
  ================================================== -->
  <!-- Themefisher Icon font -->
  <link rel="stylesheet" href="plugins/themefisher-font/style.css">
  <!-- bootstrap.min css -->
  <link rel="stylesheet" href="plugins/bootstrap/css/bootstrap.min.css">
  <!-- Lightbox.min css -->
  <link rel="stylesheet" href="plugins/lightbox2/dist/css/lightbox.min.css">
  <!-- animation css -->
  <link rel="stylesheet" href="plugins/animate/animate.css">
  <!-- Slick Carousel -->
  <link rel="stylesheet" href="plugins/slick/slick.css">
  <!-- Main Stylesheet -->
  <link rel="stylesheet" href="css/style.css">  
  
  <script src="https://kit.fontawesome.com/5ce389c680.js" crossorigin="anonymous"></script>

</head>

<body id="body">

 <!-- Start Preloader
  ==================================== -->
  <div id="preloader">
    <div class='preloader'>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
    </div>
  </div> 
<!-- End Preloader
  ==================================== -->

<!-- Fixed Navigation
  ==================================== -->
<header class="navigation fixed-top">
  <div class="container">
    <!-- main nav -->
    
    <nav class="navbar navbar-expand-lg navbar-light">
      <!-- logo -->
      <a class="navbar-brand logo" href="index.html">
        <img class="logo-default" src="images/recomb-logo/logo-name.png" alt="RECOMB Conference Logo"/>
        <img class="logo-white" src="images/recomb-logo/logo-name-white.png" alt="RECOMB Conference Logo"/>
      </a>
      <!-- /logo -->
      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
        aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
        <span class="navbar-toggler-icon"></span>
      </button>

      <div class="collapse navbar-collapse" id="navigation">
        <ul class="navbar-nav mr-auto text-center">
          <li class="nav-item ">
            <a class="nav-link" href="index.html">Home</a>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Conference</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="committees.html">Committees</a>
              <a class="dropdown-item" href="keydates_and_deadlines.html">Key dates & Deadlines</a>
              <a class="dropdown-item" href="call_for_papers.html">Call for Papers</a>
              <a class="dropdown-item" href="call_for_abstracts.html">Call for Abstracts</a>
              <a class="dropdown-item" href="call_for_highlights.html">Call for Highlights</a>
              <!-- <a class="dropdown-item" href="program.html">Program</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Register</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="register.html">RECOMB and Satellites</a>
              <a class="dropdown-item" href="travel_fellowships.html">Travel Fellowships</a>
              <!-- <a class="dropdown-item" href="accepted_papers.html">Accepted Papers</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Program</a>
            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="program.html">Program Schedule</a>
              <a class="dropdown-item" href="keynotes.html">Keynotes</a>
              <a class="dropdown-item" href="posters.html">Poster Guidelines</a>
              <!-- <a class="dropdown-item" href="accepted_papers.html">Accepted Papers</a> -->
              <!-- <a class="dropdown-item" href="call_for_abstracts.html#submission">Travel Fellowship</a> -->
              <!-- <a class="dropdown-item" href="posters.html">Posters</a> -->
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Venue</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="venue_main.html">Main Conference Venue</a>
              <a class="dropdown-item" href="venue_satellites.html">Satellites Venue</a>
              <a class="dropdown-item" href="hotels.html">Hotels</a>
              <a class="dropdown-item" href="airport.html">Airport Directions</a>
	      <a class="dropdown-item" href="things_to_do.html">Things to do @İstanbul</a>
              <a class="dropdown-item" href="transportation.html">Local Transportation</a>
	      <a class="dropdown-item" href="history_istanbul.html">History of İstanbul</a>
	      <a class="dropdown-item" href="visa.html">Visa Information</a>
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Satellites</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="https://recomb-seq.github.io/" target="_blank">RECOMB-Seq</a>
              <a class="dropdown-item" href="https://sites.google.com/view/recomb-ccb2023" target="_blank">RECOMB-CCB</a>
              <a class="dropdown-item" href="https://recomb-cg.org/" target="_blank">RECOMB-CG</a>
              <a class="dropdown-item" href="recombgenetics.html">RECOMB-Genetics</a>
            </div>
          </li>
          <li class="nav-item dropdown active">
            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
              aria-haspopup="true" aria-expanded="false">Sponsors</a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdown">
              <a class="dropdown-item" href="sponsors.html">RECOMB Sponsors</a>
              <a class="dropdown-item" href="sponsorship_packages.html">Sponsorship Packages</a>
            </div>
          </li>
          <li class="nav-item ">
            <a href="https://www.recomb.org" target="blank" style="color:white;">About RECOMB</a>
          </li>
        </ul>
      </div>
    </nav>
    <!-- /main nav -->
  </div>
</header>
<!-- End Fixed Navigation
  ==================================== -->

<section class="single-page-header">
	<div class="container">
		<div class="row">
			<div class="col-md-12">
				<h2>Poster Guidelines</h2>
			</div>
		</div>
	</div>
</section>


<!-- Start Poster Guidelines
    ==================================== -->
<section class="team-skills section-sm" id="skills">
  <div class="container">
    <div class="col-12">
      <div class="team-skills-content center">

        <hr> <h2>Posters</h2> <hr>
        
        Posters should be up to 48 inches wide by 36 inches tall (121.92 x 91.44 cm), printed in landscape mode. Please print your posters in advance. <br><br>
        
                <hr> <h4>Accepted Posters</h4> <hr>
	<a href="https://docs.google.com/document/d/1OK2oSsDjQYuo20jcJXwf8H5H_PMF6t_MOWO9QZOkFPg/edit?usp=sharing" target="_blank">Click here for the abstracts</a>
        
        <ul>
        <li>Mikaela Koutrouli, Pau Piera Lindez, Robbin Bouwmeester, Lennart Martens and Lars Juhl Jensen. "FAVA: High-quality functional association networks inferred from scRNA-seq and proteomics data".</li>
<li>Parth Pandit and Dr Mushtaque Shaikh. "Stressometer – a point of care diagnostics tool for rapid cortisol detection".</li>
<li>Sara Mohammad Taheri, Vartika Tewari, Rohan Kapre, Ehsan Rahiminasab, Karen Sachs, Charles Tapley Hoyt, Jeremy Zucker and Olga Vitek. "Experimental design for causal query estimation in partially observed biomolecular networks".</li>
<li>Fuzhou Wang, Tingxiao Gao, Jiecong Lin, Zetian Zheng, Lei Huang, Muhammad Toseef, Xiangtao Li and Ka-Chun Wong. "GILoop: Towards the graph-image duality of Hi-C data".</li>
<li>Gagandeep Singh, Mohammed Alser, Alireza Khodamoradi, Kristof Denolf, Can Firtina, Meryem Banu Cavlak, Henk Corporaal and Onur Mutlu. "RUBICON: A Framework for Designing Efficient Deep Learning-Based Genomic Basecallers".</li>
<li>Shivam Kumar, Dipanka Tanu Sarmah, Shailendra Asthana and Samrat Chatterjee. "konnect2prot: a web application to explore the protein properties in a functional protein–protein interaction network".</li>
<li>Zhihang Hu, Qinze Yu, Yucheng Guo, Taifeng Wang, Irwin King, Xin Gao, Le Song and Yu Li. "Drug Synergistic Combinations Predictions via Large-Scale Pre-Training and Graph Structure Learning".</li>
<li>Pritika Ramharack, Kwandile Mbhele, Oelfah Patel and Rabia Johnson. "Characterizing the Pharmacophoric Features of Bufadienolides from Drimia species as Na+/K+ ATPase Inhibitors: Advancing Cancer Therapeutics through Molecular Dynamic Simulations".</li>
<li>Yongjian Yang, Guanxun Li, Bo-Jia Chen and James Cai. "GenKI: a variational graph autoencoder-based virtual knockout tool for gene function prediction via single-cell gene regulatory network perturbation".</li>
<li>Ziyi Mo and Adam Siepel. "Domain adaptive neural networks improve robustness of supervised machine learning models based on simulated population genetic data".</li>
<li>Daria Emekeeva, Max Brazhnikov, Anna Kopeykina, Mikhail Gorshkov, Irina Tarasova, Lev Levitsky and Mark Ivanov. "Mapping SNP and alternative splicing events to post-translational modifications in Alzheimer disease".</li>
<li>Elizaveta Kazakova, Mark Ivanov, Tomiris Kusainova, Julia Bubis, Vladimir Gorshkov, Yanina Delegan, Inna Solyanikova, Frank Kjeldsen, Mikhail Gorshkov and Irina Tarasova. "Rapid quantitative characterization of microbial communities using ultrafast MS/MS-free proteomics".</li>
<li>Lucia Trastulla and Francesco Iorio. "Multi-omic reconciliation of in-vitro cancer cell lines and primary tumour samples".</li>
<li>Kang Jin, Zuobai Zhang, Bruce Aronow and Jian Shu. "Gene colocalization-aware segmentation of image-based spatial transcriptomics using Graph Neural Networks".</li>
<li>E. Ravza Gur, Simone G. Riva and Jim R. Hughes. "scATAC-seq identifies regulatory elements more sensitively compared to bulk ATAC-seq ".</li>
<li>Sylvain Moinard, Edouard Oudet, Didier Piau, Eric Coissac and Christelle Gonindard-Melodelima. "The Fixed Landscape Inference MethOd (flimo): an alternative to Approximate Bayesian Computation, faster by several orders of magnitude".</li>
<li>Juris Viksna, Karlis Cerans, Lelde Lace and Gatis Melkus. "Hybrid system model for haematopoietic cell differentiation".</li>
<li>Timo Westerdijk, Gizem Taş, Jan Herman Veldink, Alexander Schönhuth and Marleen Balvert. "Non-linear Dimensionality Reduction of Whole-Genome Sequencing (WGS) Data using Autoencoders".</li>
<li>William Yashar, Garth Kong, Jake Vancampen, Brittany Curtiss, Daniel Coleman, Lucia Carbone, Galip Yardimci, Julia Maxson and Theodore Braun. "GoPeaks: histone modification peak calling for CUT&Tag".</li>
<li>Joseph Estabrook, William Yashar, Hannah Holly, Julia Somers, Olga Nikolova, Ozgun Babur, Theodore Braun and Emek Demir. "Predicting transcription factor activity using prior biological information".</li>
<li>Francesca Petralia, Azra Krek, Anna Calinawan, Daniel Charytonowicz, Robert Sebra, Song Feng, Sara Gosline, Pietro Pugliese, Michele Ceccarelli and Pei Wang. "BayesDeBulk: A Flexible Bayesian Algorithm for the Deconvolution of Bulk Tumor Data".</li>
<li>Tomiris Kusainova, Daria Emekeeva, Elizaveta Kazakova, Vladimir Gorshkov, Frank Kjeldsen, Irina Ol'Khovskaya, Mikhail Kuskov, Alexey Zhigach, Natalia Gluschenko, Olga Bogoslovskaya and Irina Tarasova. "Iron nanoparticles improve growth of Triticum aestivum L. seedlings due to enhanced metabolism activity".</li>
<li>Gatis Melkus, Andrejs Sizovs, Sandra Siliņa, Peteris Rucevskis, Lelde Lace, Juris Viksna and Edgars Celms. "Clique-based identification of functional modules in Hi-C graphs".</li>
<li>Conor Walker and Gamze Gürsoy. "Private information leakage in single-cell omics data".</li>
<li>Armaghan Sarvar, Lauren Coombe, René Warren and Inanc Birol. "Genome misassembly detection using Stash: A data structure based on stochastic tile hashing".</li>
<li>René L Warren, Michael J Allison, M. Louie Lopez, Neha Acharya-Patel, Lauren Coombe, Cecilia L. Yang, Caren Helbing and Inanc Birol. "Unikseq: Unique region identification in genome sequences using a k-mer approach, to empower environmental DNA assay design and comparative genomics studies".</li>
<li>Matteo Serra, Nicola Occelli, Frederic Lifrange, Mattia Rediti, Xiaoxiao Wang, Delphine Vincent, Ghizlane Rouas, Ligia Craciun, Denis Larsimont, Miikka Vikkula, François Duhoux, David Venet, Françoise Rothé and Christos Sotiriou. "Deciphering microenvironment heterogeneity in luminal breast cancer by combining spatial transcriptomics, single cell RNA sequencing and image analysis".</li>
<li>Andrejs Sizovs, Sandra Silina, Gatis Melkus, Peteris Rucevskis, Edgars Celms, Lelde Lace and Juris Viksna. "Visualisation and exploration tools for chromatin networks".</li>
<li>Emel Comak, Stefan Haas and Martin Vingron. "Identifying enhancer hijacking events in cancer".</li>
<li>Samuele Manessi, Matteo Barcella and Ivan Merelli. "scVAR: a tool for integration of genomics and transcriptomics in scRNA-seq".</li>
<li>Zahra Nozari, Marian Schön, Jakob Simeth and Rainer Spang. "Combining Data from Different Technologies in Digital Tissue Deconvolution - a Machine Learning Approach".</li>
<li>Seungwan Hong, Yoolim A. Choi, Daniel Joo and Gamze Gursoy. "Privacy-preserving prediction of phenotypes from genotypes using homomorphic encryption".</li>
<li>Euxhen Hasanaj, Amir Alavi, Anupam Gupta, Barnabás Póczos and Ziv Bar-Joseph. "Multiset multicover methods for discriminative marker selection".</li>
<li>Jiangtao Liang and Igor Sharakhov. "Molecular organization and evolution of pericentromeric heterochromatin in mosquito species of the Anopheles gambiae complex".</li>
<li>Burcu Gurun, Jeffrey W. Tyner, Emek Demir, Brian J. Druker and Paul T. Spellman. "Computational modeling of methylation impact of AML driver events".</li>
<li>Yao-Zhong Zhang, Zeheng Bai and Seiya Imoto. "Dysfunctional analysis of the pre-training model on nucleotide sequences and the evaluation of different k-mer embeddings".</li>
<li>Kristina Thedinga and Ralf Herwig. "Combining gradient tree boosting and network propagation for the identification of a pan-cancer survival network".</li>
<li>Kiwamu Arakane, Hiroaki Imoto and Mariko Okada. "Applying Text-Mined Knowledge from PubMed and PubMed Central to Signaling Pathways".</li>
<li>Abdulkadir Elmas and Kuan-Lin Huang. "Expression-Driven Dependency Analyses Identify New Cancer Vulnerabilities".</li>
<li>Barbara Höllbacher, Ming Wu, Christine Wurmser, Talyn Chu, Dietmar Zehn and Matthias Heinig. "Multi-omics single-cell analyses reveal precursors of exhausted T-cells are preemptively formed regardless of the outcome of infection".</li>
<li>Myvizhi Esai Selvan, Kenan Onel, Sacha Gnjatic, Robert Klein and Zeynep Gümüş. "Inherited Rare Deleterious Variants in Cancer Risk: A Large-Scale Pan-Cancer Case-Control Analysis".</li>
<li>Valeriy Postoenko, Mark Ivanov, Julia Bubis, Anastasiya Lipatova, Daria Emekeeva, Irina Tarasova and Mikhail Gorshkov. "Ultrafast label-free time series proteome profiling using DirectMS1 approach".</li>
<li>Aybuge Altay and Martin Vingron. "Annotating scATAC-seq clusters with bulk ATAC-seq data".</li>
<li>Yan Zhao, Clemens Kohl and Martin Vingron. "CAbiNet: Joint visualization of cells and genes based on a gene-cell graph".</li>
<li>Kami Pekayvaz, Corinna Losert, Viktoria Knottenberg, Matthias Heinig, Leo Nicolai and Konstantin Stark. "Multi-Omic Factor Analysis uncovers immunologic signatures with pathophysiologic and clinical implications in coronary syndromes ".</li>
<li>Adrian Fritz and Alice McHardy. "CAMISIM 2.0".</li>
<li>Evelin Aasna, Bjarni Halldórsson, Snaedís Kristmundsdóttir and Guillaume Holley. "De novo variant detection".</li>
<li>Mitra Darja Darvish, Enrico Seiler, Svenja Mehringer, René Rahn and Knut Reinert. "Fast and space-efficient indices for very large collections of sequencing experiments".</li>
<li>Rosemary He, Varuni Sarwal, Jingyuan Fu, Valerie Arboleda, Eleazar Eskin and Kenny Qian. "RiVAD: a pipeline for respiratory virus agnostic testing".</li>
<li>Damla Senol Cali, Konstantinos Kanellopoulos, Joel Lindegger, Zülal Bingöl, Gurpreet Singh Kalsi, Ziyi Zuo, Can Firtina, Meryem Banu Cavlak, Jeremie S. Kim, Nika Mansouri Ghiasi, Gagandeep Singh, Juan Gómez Luna, Nour Almadhoun Alserr, Mohammed Alser, Sreenivas Subramoney, Can Alkan, Saugata Ghose and Onur Mutlu. "SeGraM: A Universal Hardware Accelerator for Genomic Sequence-to-Graph and Sequence-to-Sequence Mapping".</li>
<li>Jingyuan Fu and Eleazar Eskin. "ImputeNN: a deep-learning autoencoder tool for genotype imputation".</li>
<li>Kun-Hyung Roh and Charles Mobbs. "Computational pharmacogenomics to uncover targets of phenothiazine derivatives to reverse Alzheimer’s Disease pathology".</li>
<li>Siyu Han and Rui Wang-Sattler. "Missing Panel Imputation for Longitudinal Omics Data Using Representation Disentanglement and Time Point Transfer".</li>
<li>Natalia Rutecka and Pawel Gorecki. "Robinson-Foulds distance between phylogenetic networks and gene trees".</li>
<li>Jiangtao Liang, Noah Rose, Ilya Brusentsov, Varvara Lukyanchikova, Dmitriy Karagodin, Zhijian Tu, Carolyn McBride and Maria Sharakhova. "Chromosomal inversions are associated with geographical origin, behavior, and resistance to pathogens in the mosquito Aedes aegypti".</li>
<li>Yosef Masoudi-Sobhanzadeh, Shuxiang Li, Yunhui Peng and Anna R Panchenko. "Identifying nucleosome positioning features based on Deep Residual Networks ".</li>
<li>Haiyu Mao, Mohammed Alser, Mohammad Sadrosadati, Can Firtina, Akanksha Baranwal, Damla Senol Cali, Aditya Manglik, Nour Almadhoun Alserr and Onur Mutlu. "GenPIP: In-Memory Acceleration of Genome Analysis via Tight Integration of Basecalling and Read Mapping".</li>
<li>Can Firtina, Nika Mansouri Ghiasi, Joel Lindegger, Gagandeep Singh, Meryem Banu Cavlak, Haiyu Mao and Onur Mutlu. "RawHash: Enabling Fast and Accurate Real-Time Analysis of Raw Nanopore Signals for Large Genomes".</li>
<li>Can Firtina, Kamlesh Pillai, Gurpreet S. Kalsi, Bharathwaj Suresh, Damla Senol Cali, Jeremie S. Kim, Taha Shahroodi, Meryem Banu Cavlak, Joel Lindegger, Mohammed Alser, Juan Gómez-Luna, Sreenivas Subramoney and Onur Mutlu. "ApHMM: Accelerating Profile Hidden Markov Models for Fast and Energy-Efficient Genome Analysis".</li>
<li>Jeremie S. Kim, Can Firtina, Meryem Banu Cavlak, Damla Senol Cali, Nastaran Hajinazar, Mohammed Alser, Can Alkan and Onur Mutlu. "AirLift: A Fast and Comprehensive Technique for Remapping Alignments between Reference Genomes".</li>
<li>Ecem Ültanır, İrem Urhan, Şule Erkul and Arzucan Özgür. "Identifying viral-mediated pathogenic pathways in ALS disease using PPI network-based approach".</li>
<li>Mohammad Lotfollahi, Carlo De Donno, Soroor Hediyeh-Zadeh, Marco Wagenstetter, Amir Ali Moinfar, Luke Zappia and Fabian J. Theis. "Population-level integration of single-cell datasets enables multi-scale analysis across samples".</li>
<li>Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu. "TargetCall: Eliminating the Wasted Computation in Basecalling via Pre-Basecalling Filtering".</li>
<li>Shadi Darvish Shafighi, Barbara Jurzysta and Ewa Szczurek. "ClonalGE: Clonal gene expression analysis from spatial transcriptomics data".</li>
<li>Alex Lederer, Lorenzo Talamanca, Colas Droin, Maxine Leonardi, Irina Khven, Hugo Carvalho, Felix Naef and Gioele La Manno. "A probabilistic framework for parametrizing RNA velocity fields with manifold-consistent cell cycle dynamics".</li>
<li>Chakravarthi Kanduri, Lonneke Scheffer, Milena Pavlović, Knut Dagestad Rand, Maria Chernigovskaya, Oz Pirvandy, Gur Yaari, Victor Greiff and Geir Kjetil Sandve. "simAIRR: simulation of adaptive immune repertoires with realistic receptor sequence sharing for benchmarking of immune state prediction methods".</li>
<li>Leyla A. Garibova, Valeriy I. Postoenko, Lev I. Levitsky, Julia A. Bubis, Mikhail V. Gorshkov and Mark V. Ivanov. "IQMMA: an efficient MS1 intensity extraction using multiple feature detection algorithms for DDA proteomics ".</li>
<li>Joel Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez Luna, Nika Mansourighiasi and Onur Mutlu. "Scrooge: A Fast and Memory-Frugal Genomic Sequence Aligner for CPUs, GPUs, and ASICs".</li>
<li>Catherine Mahoney, Michael Salter-Townshend and Denis Shields. "Detecting genomic regions enriched for recombination in autism spectrum disorder".</li>
<li>Mihir Bafna, Hechen Li and Xiuwei Zhang. "CLARIFY: Cell-cell interaction and gene regulatory network refinement from spatially resolved transcriptomics".</li>
<li>Jazeps Medina Tretmanis and Emilia Huerta-Sanchez. "Joint calling of local ancestry and archaic introgression using deep learning.".</li>
<li>Kitti Banga and Donovan Bailey. "Examining the Accessible Chromatin Regions Between Duplicated Gene Copies in the Tetraploid Leucaena trichandra".</li>
<li>Ali Karakoc, Ege Ezen and Tayfun Tatar. "Discovery of Novel Glioblastoma Markers from Exosomal miRNAs & Cross-linking to Tissue Expression Profiles".</li>
<li>Ekin Deniz Aksu and Martin Vingron. "Primate enhancers: identification, annotation, evolution".</li>
<li>Ruihua Zhang and Gesine Reinert. "Simulating weak attacks in a new duplication-divergence model with gene loss".</li>
<li>Palash Sashittal, Haochen Zhang, Christine A. A. Iacobuzio-Donahue and Ben Raphael. "ConDoR: Tumor phylogeny inference with a copy-number constrained mutation loss model".</li>
<li>Myeong Gyu Kim and Kyoung Eun Kim. "Prediction of drug concentration using machine learning algorithms".</li>
<li>Valentina Giansanti, Oronzina Botrugno, Francesca Giannese, Dejan Lazarević, Giovanni Tonon, Marco Antoniotti and Davide Cittaro. "MOWGAN: a multiomic single cell data integration framework to dissect deep biology".</li>
<li>Xin Lai. "A network medicine approach for identifying diagnostic and prognostic biomarkers and exploring drug repurposing in human cancer".</li>
<li>Michał Kowalski, Kamila Marszałek, Wojciech Branicki and Paweł P. Łabaj. "2015 called and wants its technology back – what metagenomics can gain from machine learning techniques which are not trendy anymore".</li>
<li>Gözde Yazıcı, Burcu Kurt Vatandaşlar, İlknur Aydın Cantürk, Fatmagül İlayda Aydınlı, Özge Arıcı Düz, Emre Karakoç, Bilal Ersen Kerman and Can Alkan. "Identification of protein-protein interaction bridges for multiple sclerosis".</li>
<li>Mike Lopez III, Donovan Bailey and Katie Banga. "Using Bisulfite Sequencing to Characterize the Fate of Duplicate Gene Copies ".</li>
<li>Nicholas Hutchins and Pulin Li. "Looking into the history of cells: transferring cross-cell type functional information".</li>
<li>Katarzyna Sidorczuk, Przemysław Gagat, Jakub Kała, Henrik Nielsen, Filip Pietluch, Paweł Mackiewicz and Michal Burdukiewicz. "Where do they come from, where do they go? Prediction of protein subplastid localization  and origin with PlastoGram".</li>
<li>Katarzyna Sidorczuk, Przemyslaw Gagat, Filip Pietluch, Jakub Kała, Dominik Rafacz, Laura Bąkała, Jadwiga Słowik, Rafał Kolenda, Stefan Rödiger, Legana Fingerhut, Ira Cooke, Paweł Mackiewicz and Michal Burdukiewicz. "Benchmarks in antimicrobial peptide prediction are biased due to the selection of negative data".</li>
<li>Stavros Makrodimitris, Bram Pronk, Tamim Abdelaal and Marcel Reinders. "A critical comparison of linear and non-linear joint embedding methods for bulk and single-cell multi-omics".</li>
<li>Michelle Almeida da Paz, Sarah Warger and Leila Taher. "Standard high-throughput functional analysis pipelines omit specific repetitive genetic groups".</li>
<li>Irem B. Gündüz, Bei Wei, Fabian Müller and William Greenleaf. "Chromatin accessibility landscape of human peripheral blood cells upon pathogen exposure".</li>
<li>Cem Güleç and Matthias Heinig. "Enabling Cell Atlas Guided Optimal Experimental Design".</li>
<li>Tobias Schmidt, Rainer Spang, Wolfram Gronwald and Michael Altenbuchinger. "Explainable AI for the molecular subclassification of DLBCL".</li>
<li>Gözde Kibar and Martin Vingron. "Prediction of protein-protein interactions using sequences of intrinsically disordered regions".</li>
<li>Begum Ozemek, Hasan Ozan Otaş and Marie-Laure Yaspo. "Systematic analysis of enhancer RNAs for understanding enhancer-mediated gene expression regulation".</li>
<li>Gözde Atağ, Kıvılcım Başak Vural, Damla Kaptan, Mustafa Özkan, Dilek Koptekin, Ekin Sağlıcan, Sevcan Doğramacı, Mevlüt Köz, Ardan Yılmaz, Arda Söylev, İnci Togan, Mehmet Somel and Füsun Özer. "MTaxi: A comparative tool for taxon identification of ultra low coverage ancient genomes".</li>
<li>Daria Ostroverkhova, Kathrin Tyryshkin, Igor Rogozin, Konstantin Shaitan, Polina Shcherbakova and Anna Panchenko. "Mutated DNA Polymerase Epsilon Generates Distinct Mutational Landscape in Endometrial Cancer Genomes".</li>
<li>Jiří Novotný, Jan Kubovčiak and Michal Kolář. "scdrake: a reproducible and scalable pipeline for scRNA-seq data analysis".</li>
<li>Merve Nur Güler, Tara Ekin Ünverdi, Ardan Yılmaz, Igor Mapelli, Kıvılcım Başak Vural, Kanat Gürün, Emre Akbaş and Mehmet Somel. "DISTINGUISHING FIRST-DEGREE RELATIONSHIPS FROM ANCIENT SAMPLES USING MACHINE LEARNING ".</li>
<li>Marco Anteghini, Chethan K. Krishna, Vishal Kalel and Edoardo Saccenti. "Computational approaches for localizing peroxisomal proteins: integrating Deep Learning (DL)-based embbeddings and Peroxisome Targeting Signal information.".</li>
<li>Liangti Dai and Gerton Lunter. "EpiCall: A high-resolution peak caller for ATAC-seq data".</li>
<li>Daniel Espiritu, Yunhui Peng, Nadejda Boev, David Landsman, Maria Aristizabal and Anna Panchenko. "Elucidating the Impact of Histone Mutations in Cancer".</li>
<li>Gian Marco Messa, Peng Liu, Francesco Napolitano, Xin Gao and Valerio Orlando. "Dynamic simulation of whole body and organ to organ communication using metabolic networks".</li>
<li>Roan Zaied, Tayaza Fadason and Justin O'Sullivan. "De novo identification of complex traits associated with asthma".</li>
<li>Intekhab Hossain, John Quackenbush and Rebekka Burkholz. "Biologically informed NeuralODEs for genome-wide regulatory dynamics".</li>
<li>Piotr Sliwa, Heather Harrington, Gesine Reinert and Julian C. Knight. "Community Detection Analysis in Multilayer COVID-19 and sepsis Patient Similarity Networks".</li>
<li>Aysun Urhan, Bianca-Maria Cosma, Abigail L. Manson, Ashlee M. Earl and Thomas Abeel. "SAP: Synteny-aware gene function prediction for bacteria using protein embeddings".</li>
<li>Nika Mansourighiasi, Jisung Park, Harun Mustafa, Jeremie Kim, Ataberk Olgun, Arvid Gollwitzer, Damla Senol Cali, Can Firtina, Haiyu Mao, Nour Almadhoun Alserr, Rachata Ausavarungnirun, Nandita Vijaykumar, Mohammed Alser and Onur Mutlu. "GenStore: A High-Performance In-Storage Processing System for Genome Sequence Analysis".</li>
<li>Serena Hughes, Timothy Hamilton and Eric Deeds. "A Statistical Marker Gene Clustering Tool".</li>
<li>Knut Rand, Ivar Grytten, Milena Pavlovic, Chakravarthi Kanduri and Geir Kjetil Sandve. "BioNumPy: Fast and easy analysis of biological data with Python".</li>
<li>Lena Morrill Gavarró and Florian Markowetz. "Rise and fall of mutational signatures: a simulation study".</li>
<li>Milena Pavlovic, Lonneke Scheffer, Chakravarthi Kanduri, Maria Chernigovskaya, Knut Rand, Ivar Grytten, Victor Greiff and Geir Kjetil Sandve. "immuneML: an open-source platform for machine learning analysis of adaptive immune receptors and repertoires".</li>
<li>Asu Büşra Temizer, Rıza Özçelik, Taha Koulani, Gökçe Uludoğan, Elif Ozkirimli, Kutlu O. Ulgen, Nilgün Karalı and Arzucan Özgür. "Interpreting Chemical Words of a Data-driven Segmentation Method as Protein Family Pharmacophores and Functional Groups".</li>
<li>Neeraja Krishnan, Sarah Rahman and Binay Panda. "X-plat: a cross-platform transformation tool based on the nearest neighbor joining framework.".</li>
<li>Maria Chernigovskaya, Milena Pavlović, Chakravarthi Kanduri, Geir Kjetil Sandve and Victor Greiff. "Statistical analysis of synthetic AIRR-datasets to guide the development and benchmarking of AIRR-based machine learning ".</li>
<li>Simon Wengert, Xenofon Giannoulis, Peter Kreitmeier, Matthias Heinig and Na Cai. "Investigating tissue-specific apparent mtDNA heteroplasmy and its relationship with ageing and mtDNA gene expression".</li>
<li>Ekaterina Kazantseva, Ataberk Donmez, Mihai Pop and Mikhail Kolmogororov. "An assembly-based approach to metagenomic strain phasing using long reads".</li>
<li>Pasquale Luca Curci, Nicoletta Rapanà and Mariella Finetti-Sialer. "Transcriptomic profiles in a biostimulant-plant-pathogen interaction: The Capsicum annuum system".</li>
<li>Luigi Laezza, Luigi Ferraro and Michele Ceccarelli. "Embedding protein interactions with graph attention network: Proteins behaviours in anticancer drugs synergy".</li>
<li>Nicole Zatorski and Avner Schlessinger. "Characterization of Withdrawn Drugs Using an Ensemble Approach on Molecular and Target Features".</li>
<li>Gökçe Uludoğan, Elif Ozkirimli, Kutlu Ö. Ülgen, Nilgün Karalı and Arzucan Ozgur. "A Unified Framework for Target-Based Drug Discovery with Multimodal Contrastive Learning".</li>
<li>Travis Moore, Ryan Mulqueen, Kevin McPherson, Andrew Adey, Hisham Mohammed and Galip Yardimci. "Robust CNV detection using single-cell ATAC-seq".</li>
<li>Lokesh Thangamani, Murugesh Eswaran, Yamuna Annadurai, Jeyakumar Natarajan and Shanmughavel Piramanayagam. "Bare metal instances from Cloud computing for the establishment of microbiota in  Alzheimer’s Gut-Brain axis and its dysbiosis".</li>
<li>Yamuna Annadurai, Murugesh Easwaran, Lokesh Thangamani, Jeyakumar Natarajan and Shanmughavel Piramanayagam. "Artificial Neural Network specific ML algorithm for monomeric protein artifacts using NEAT and TDNN".</li>
<li>Joke Deschildre, Boris Vandemoortele, Jens Uwe Loers, Katleen De Preter and Vanessa Vermeirssen. "Evaluation of single-sample network inference methods for precision oncology".</li>
<li>Cansu Damla Yılmaz, Gökçe Uludoğan, Arzucan Özgür, Elif Ozkirimli, Asu Büşra Temizer and Nilgün Karalı. "Effect of Protein and Drug Words in Drug-Target Affinity Prediction".</li>
<li>Carla Castignani, Jonas Demeulemeester, Peter Van Loo, Charles Swanton, Robert E. Hynds and Nnenna Kanu. "CREDAC: Copy number-based Reference-free Expression Deconvolution Analysis of Cancers ".</li>
<li>Sevilay Güleşen, Clara T. Schoeder and Jens Meiler. "B-cell Epitope Prediction of Marburg Virus Glycoprotein through Accelerated Class I Fusion Protein Epitope Mapping (AxIEM)".</li>
<li>Mallika Varkhedi, Vayda Barker and George Blanck. "CNV assessments of MAPT in Neuroblastoma and correlations with survival probability".</li>
<li>Henri Schmidt, Minsi Zhang, Dimitar Chakarov, Vineet Bansal, Haralambos Mourelatos, Francisco Sánchez-Rivera, Scott Lowe, Andrea Ventura, Christina Leslie and Yuri Pritykin. "Genome-wide CRISPR guide RNA design and specificity analysis with GuideScan2".</li>
<li>Tom Ouellette, Yiran Shao and Philip Awadalla. "Got segments? Or how to maximize phenotypic insights from segmented cellular images".</li>
<li>Harun Mustafa, Mikhail Karasikov, Gunnar Rätsch and André Kahles. "MetaGraph-MLA: Label-guided alignment to variable-order De Bruijn graphs".</li>
<li>Lukasz Peplowski. "Application of theoretical computational biophysics methods for improving biotechnological enzyme Nitrile Hydratase".</li>
<li>Karolina Mikulska-Ruminska. "Studies on the inhibitory mechanism of the ferroptotic cell death signal induced by lipoxygenases".</li>

        </ul>
        
      </div>
    </div>    
  </div> <!-- End container -->
</section> <!-- End section -->
<!-- End Poster Guidelines
    ==================================== -->



<!-- begin Footer Area
    ========================================== -->
<footer id="footer" class="bg-one">
  <div class="top-footer">
    <p>Any questions? Email organizers at <b>recomb2023@bilkent.edu.tr</b></p>
  </div>
  <div class="footer-bottom">
    <h5>Copyright 2023. All rights reserved.</h5>
    <h6>Design and Developed by Themefisher</h6>
  </div>
</footer> <!-- end footer -->
<!-- end Footer Area
    ========================================== -->
    
<!-- Essential Scripts
  =====================================-->
<!-- Main jQuery -->
<script src="plugins/jquery/jquery.min.js"></script>
    
<!-- Bootstrap4 -->
<script src="plugins/bootstrap/js/bootstrap.min.js"></script>

<!-- lightbox -->
<script src="plugins/lightbox2/dist/js/lightbox.min.js"></script>

<!-- Owl Carousel -->
<script src="plugins/slick/slick.min.js"></script>
    
<!-- Custom js -->
<script src="js/script.js"></script>

<script>
  $(document).ready(function() {
  
  // Add minus icon for collapse element which is open by default
  $(".collapse.show").each(function() {
    $(this).prev(".card-header").find(".fas").addClass("fa-angle-double-down").removeClass("fa-angle-double-down");
  });

  // Toggle plus minus icon on show hide of collapse element
  $(".collapse").on('show.bs.collapse', function() {
    $(this).prev(".card-header").find(".fas").removeClass("fa-angle-double-down").addClass("fa-angle-double-up");}).on('hide.bs.collapse', function(){
      $(this).prev(".card-header").find(".fas").removeClass("fa-angle-double-up").addClass("fa-angle-double-down");
    });
  });
</script>

<!-- Default Statcounter code for RECOMB2023 http://recomb2023.bilkent.edu.tr -->
<script type="text/javascript">
  var sc_project=12524244;
  var sc_invisible=1;
  var sc_security="f3f6312a";
</script>

<script type="text/javascript" src="https://www.statcounter.com/counter/counter.js"async></script>

<noscript><div class="statcounter"><a title="Web Analytics"
  href="https://statcounter.com/" target="_blank"><img
  class="statcounter"
  src="https://c.statcounter.com/12524244/0/f3f6312a/1/"
  alt="Web Analytics"></a></div>
</noscript>
<!-- End of Statcounter Code -->

</body>
</html>
